src/sentry/integrations/aws_lambda/integration.py

```diff
@@ -379,7 +379,7 @@ def _enable_lambda(function):
         failures = []
         success_count = 0
 
-        with ThreadPoolExecutor(max_workers=10) as _lambda_setup_thread_pool:
+        with ThreadPoolExecutor(max_workers=100) as _lambda_setup_thread_pool:
```

Comment:
Can we make it a setting instead of hardcoding the value?

Reply:
@beezz fixed